src/init/inst-functions.sh

```diff
@@ -1033,6 +1033,11 @@ InstallServer()
     ${INSTALL} -m 0750 -o root -g ${OSSEC_GROUP} ../wodles/aws/aws_s3.py ${PREFIX}/wodles/aws/aws-s3.py
     ${INSTALL} -m 0750 -o root -g ${OSSEC_GROUP} ../framework/wrappers/generic_wrapper.sh ${PREFIX}/wodles/aws/aws-s3
 
+    ${INSTALL} -d -m 0750 -o root -g ${OSSEC_GROUP} ${PREFIX}/wodles/gcp-pubsub
+    ${INSTALL} -m 0750 -o root -g ${OSSEC_GROUP} ../wodles/gcloud/__main__.py ${PREFIX}/wodles/gcp-pubsub/__main__.py
```

Comment:
Pending to discuss it this structure is valid. Specifications were:

![image](https://user-images.githubusercontent.com/29475387/69074998-c818d500-0a30-11ea-8950-45aae1da878a.png)

Reply:
According to this PR (https://github.com/wazuh/wazuh/pull/4261), changes to the installation process are not needed here.